components/clearbutton/stories/clearbutton.stories.js

```diff
@@ -28,6 +61,15 @@ export default {
 			},
 			control: "boolean",
 		},
+		isQuiet: {
+			name: "Quiet Styling",
```

Comment:
```suggestion
			name: "Quiet styling",
```

Minor style guide nit